js/scrollspy.js

```diff
@@ -181,14 +186,12 @@
 		    var offset = $(this.hash).offset().top + 1;
 
 //          offset - 200 allows elements near bottom of page to scroll
-			
-	    	$('html, body').animate({ scrollTop: offset - 200 }, {duration: 400, queue: false, easing: 'easeOutCubic'});
+
+              var scrollOffset = typeof options.scrollOffset !== 'undefined' ? options.scrollOffset : 200;
+              $('html, body').animate({ scrollTop: offset - scrollOffset }, {duration: 400, queue: false, easing: 'easeOutCubic'});
 			
 		  });
 		});
-		options = options || {
-			throttle: 100
```

Comment:
no need to specify this default value here, since later on there is a fallback (`var throttledScroll = throttle(onScroll, options.throttle || 100);`) either way. a default value of `{}` will suffice